Content.Server/DeviceNetwork/Systems/DeviceNetworkSystem.cs

```diff
@@ -364,7 +365,11 @@ private void SendToConnections(ReadOnlySpan<DeviceNetworkComponent> connections,
                 if (!beforeEv.Cancelled)
                     RaiseLocalEvent(connection.Owner, packet, false);
                 else
+                {
                     beforeEv.Uncancel();
+                    var eventArgs = new SignalFailedEvent(packet.Sender); // event so signallers throw an error if they're out of range
+                    RaiseLocalEvent(packet.Sender, ref eventArgs, false);
```

Comment:
```suggestion
                    RaiseLocalEvent(packet.Sender, ref eventArgs);
```
This defaults to false